pkgs/applications/networking/instant-messengers/skypeforlinux/default.nix

```diff
@@ -4,7 +4,7 @@
 
 let
 
-  version = "5.5.0.1";
+  version = "8.10.76.2";
```

Comment:
I think it is time to add a note regarding 8.x is unstable. Every now and then somebody tries to update it.

Reply:
Sounds good.

However, 8.9.0.1 is now stable; it may outlive unstable versions in the 8.10.* series. Yet the feedback about it is hilarious:

> With the old version of skypeforlinux i didn't had any problems, when I updated it today, it started, but for some reason, after a few clicks, it goes to blank.

> After I downloaded the new version of Skype for Desktops I started getting advertisement popups.  How can I stop getting them?

> but how can I change my status ... I followed help but I cannot change the status ... no button

> I am just amazed at this haphazard, purely random changes you made, instead of trying to restore at least part of useful functionality from 4.3.

> Thank you Microsoft for continuing to work on Skype for Linux! I do appreciate the effort.
> With that said, I must say that screen sharing is still a huge issue! At least I have a button for it now, but unfortunately, it doesn't work out of the box.

https://answers.microsoft.com/en-us/skype/forum/skype_linux-skype_startms-skype_installms/the-new-skype-for-desktop-is-here/dba5e41b-126a-4221-9e28-e1266d235689

Reply:
Well something is always broken for somebody. Have you personal experience with this version?

Reply:
Jokes aside, just a few days after the release Debian [says](https://wiki.debian.org/skype#Frequently_Ask_Questions_.28F.A.Q..29_for_Stretch_64_bits) that the blank screen affects multiple users, and Microsoft explains on the linked page that this happens on CPUs without SSSE3 and they are going to fix it and that in the meantime users should stay on 5.5. So adding skypeforlinux as `skypeforlinux8` is fine, but replacing 5.5 is not.